### ISSUE TITLE
It would be good if this project could follow the gitignores for ignoring by default.

### ISSUE THREAD

**Issue by adambirds** [d-koppenhagen/vscode-file-tree-to-text-generator#5]
It would be good if this project could follow the gitignores for ignoring by default.

**Comment by d-koppenhagen** [comment]
First i thought: yes! But I re-thinked about it: The `.gitignore` file is used to not check-in code. Generating file-trees for documentation purpose is completely independent of this and the files to ignore in the generated tree are very individual depending on what you want to show. So setting this by default is not a good idea in my opinion.
Probably a boolean flag for optionally enabling a feature to respect the `.gitignore` would be cool. But on the other side there are other ignore files such as `. prettierignore` or others.

**Comment by adambirds** [comment]
Yeah I think at least an option to follow it would be good. I think that `.gitignore` would be a good one to start over others like `.prettierignore` as that one and others are more aimed at just not running linters and are not usually as long as the .gitignore files.

**Comment by adambirds** [comment]
On this, if you'd be willing to point me to the correct part of the code i'd be happy to try and work on this.
